routers/repo/view.go

```diff
@@ -66,9 +105,25 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 	// the last one is for a readme that doesn't
 	// strictly match an extension
 	var readmeFiles [4]*git.Blob
+	var docsEntries [3]*git.TreeEntry
 	var exts = []string{".md", ".txt", ""} // sorted by priority
 	for _, entry := range entries {
 		if entry.IsDir() {
+			lowerName := strings.ToLower(entry.Name())
+			switch lowerName {
+			case "docs":
+				if entry.Name() == "docs" || docsEntries[0] == nil {
+					docsEntries[0] = entry
+				}
+			case ".gitea":
+				if entry.Name() == ".gitea" || docsEntries[1] == nil {
+					docsEntries[1] = entry
+				}
+			case ".github":
+				if entry.Name() == ".github" || docsEntries[2] == nil {
+					docsEntries[2] = entry
+				}
+			}
 			continue
 		}
 
```

Comment:
I think the two blocks right below this comment (`for` and `if`) should only be executed if `entry.IsRegular()` is `true`. A symlink would probably break things.

Reply:
I don't think it would

Reply:
I've tested with a symlink, and the rendered result is the symlink contents:

```
$ ls -l
total 4
-rw-rw-r-- 1 gittest gittest 45 Feb 18 19:42 NOREAD.md
lrwxrwxrwx 1 gittest gittest  9 Feb 20 13:55 README.md -> NOREAD.md
```

![image](https://user-images.githubusercontent.com/18600385/74959182-e522c280-53e8-11ea-8027-d1bd91a9195d.png)

Reply:
@guillep2k have you try to simlink outside the repo ?

Reply:
I didn't, but a symlink to a missing file in the same dir also rendered the symlink contents (no errors). I think it will always render the linked path, rather than attempt following the link.

Reply:
I've handled symlinks now